client/renderSDL/ScreenHandler.cpp

```diff
@@ -209,12 +212,22 @@ void ScreenHandler::recreateWindowAndScreenBuffers()
 
 	initializeScreenBuffers();
 
+	setWindowIcon();
+
 	if(!settings["session"]["headless"].Bool() && settings["general"]["notifications"].Bool())
 	{
 		NotificationHandler::init(mainWindow);
 	}
 }
 
+void ScreenHandler::setWindowIcon()
+{
+	auto windowIconSurface = CSDL_Ext::newSurface(22,22);
```

Comment:
Is 22x22px enough? Considering high-DPI screens and such?

Reply:
I seached a bit and didn't find a definite answer:

https://discourse.libsdl.org/t/is-there-a-size-limit-on-the-surface-submitted-to-sdl-setwindowicon/24380

On Linux, the desktop environment mostly uses the icon specified in the d.desktop file, but maybe not on other systems?